petastorm/__init__.py

```diff
@@ -12,35 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import abc
-import six
-
 __version__ = '0.3.1'
-
-
-@six.add_metaclass(abc.ABCMeta)
-class PredicateBase(object):
```

Comment:
note this is an api breaking change. We should be releasing 0.4.0 next after this change

Reply:
yep it is.